watchtower/lookout/lookout.go

```diff
@@ -102,6 +122,49 @@ func (l *Lookout) Stop() error {
 	return nil
 }
 
+// fetchBlockWithRetries attempts to fetch a block from the blockchain using
+// its hash. If it fails to fetch the block, it will back-off and retry up to
+// MaxNumRetries times.
+func (l *Lookout) fetchBlockWithRetries(hash *chainhash.Hash) (*wire.MsgBlock,
+	error) {
+
+	backoff := l.cfg.MinBackoff
+
+	updateBackoff := func() {
+		backoff *= 2
+		if backoff > l.cfg.MaxBackoff {
+			backoff = l.cfg.MaxBackoff
+		}
+	}
+
+	var attempt int
+	for {
+		attempt++
+
+		block, err := l.cfg.BlockFetcher.GetBlock(hash)
+		if err == nil {
+			return block, nil
+		}
+
+		if attempt >= l.cfg.MaxNumRetries {
```

Comment:
nit: technically `MaxNumRetries` is rather a `MaxNumTries` right now, so having `>` instead of `>=` would be more accurate

Reply:
Done, thanks!